CMakeLists.txt

```diff
@@ -201,6 +201,13 @@ configure_file(
   @ONLY
   )
 
+set(EXTRA_WIN32_PATH)
+if(fletch_BUILT_WITH_CUDA)
+  set(EXTRA_WIN32_PATH ${EXTRA_WIN32_PATH} "${CUDA_TOOLKIT_ROOT_DIR}/bin;")
```

Comment:
Minor nit, instead of using this set pattern, it's better to do list(APPEND EXTRA_WIN32_PATH "${CUDA_TOOLKIT_ROOT_DIR}/bin;). Also it isn't necessary to set the variable to empty first. I believe it starts out that way by default.

Reply:
I appreciate any nit picks
It is also very cool that when using list, it automagically adds the OS separator for you when it outputs the value as a string!